tools/codegen/gen.py

```diff
@@ -812,6 +812,7 @@ def __call__(self, f: NativeFunction) -> Optional[str]:
                 compute_dk = f"DispatchKey _dk = {dispatch_key};"
             return f"""\
 // aten::{f.func}
+C10_ALWAYS_INLINE
```

Comment:
Are you sure this does anything? This is getting generated to a cpp file so it's hard for me to imagine that you're actually ever going to get to inline it.

Reply:
Surprisingly, yes! It can get inlined into wrap_kernel_functor_unboxed_::call; the call chain is Library::impl -> CppFunction ctor -> KernelFunction::makeFromUnboxedFunction -> KernelFunction::makeFromUnboxedFunctor -> wrap_kernel_functor_unboxed. (My working theory is that the fragility of inlining decisions for dispatcher code and the high compile times for generated aten files like Functions.cpp is the huge amount of inlineable stuff going on between this path and the call out to `Dispatcher::call{,WithDispatchKey}`, repeated for every `at::` function.)